### PR TITLE
SEO-1942_Money_Keywords_WinForms_PDFViewer

### PR DESCRIPTION
**Pull request by gowthamanthavasiyappan**
Hi @ChristoIssac,

Worked on WinForms PDF Viewer Money Keywords, merge these changes.

Regards,
Gowthaman

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,46 @@
 # WinForms-PDFViewer-Examples
 This repository contains various example projects of WinForms PDF Viewer control.
+
+In WinForms applications, the [PDF Viewer](https://www.syncfusion.com/winforms-ui-controls/pdf-viewer?utm_source=github&utm_medium=listing&utm_campaign=winforms-pdf-viewer-github-samples) control allows you to view and print PDF files. The inclusion of hyperlinks and a table of contents allows for easy navigation both within and outside of PDF files.
```
Comment (Girija-Rajendran) [inline README.md:4]: Add the platform name before the control name
[WinForms PDF Viewer](.......) control........

File: README.md [hunk 2]
```diff
@@ -1,2 +1,46 @@
...
+
+## About Syncfusion
+
+Founded in 2001 and headquartered in Research Triangle Park, N.C., Syncfusion has more than 27,000+ customers and more than 1 million users, including large financial institutions, Fortune 500 companies, and global IT consultancies.
```
Comment (Girija-Rajendran) [inline README.md:38]: more than 27,000 customers...........
remove "+"

File: README.md (hunk 1 above)
Comment (gowthamanthavasiyappan) [reply README.md]: Added WinForms before the control name

File: README.md (hunk 2 above)
Comment (gowthamanthavasiyappan) [reply README.md]: Removed '+'

**Review by Girija-Rajendran — APPROVED** [review]

**Review by ChristoIssac — APPROVED** [review]

**Pull request merged**